example/example12.cpp

```diff
@@ -52,7 +52,7 @@ class PyExample12 : public Example12 {
         PYBIND11_OVERLOAD_PURE(
             bool,
             Example12,
-            run_bool
+            run_bool,
```

Comment:
Hmm, that is pretty weird to see in the code. Do you know if there is any way to change the macro so that this warning disappears?